resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1490,7 +1490,7 @@ private[spark] object Client extends Logging {
    * Populate the classpath entry in the given environment map.
    *
    * User jars are generally not added to the JVM's system classpath; those are handled by the AM
-   * and executor backend. When the deprecated `spark.yarn.user.classpath.first` is used, user jars
+   * and executor backend. When the `spark.driver.userClassPathFirst` is used, user jars
```

Comment:
New comment is correct? It seems that we miss `spark.executor.userClassPathFirst` case.